.github/scripts/cmp-config.pl

```diff
@@ -24,76 +24,77 @@
+    '#define HAVE_LIBWOLFSSH'                          => 1,
+    '#define HAVE_LIBZSTD 1'                           => 1,
+    '#define HAVE_MSH3_H 1'                            => 1,
+    '#define HAVE_NGHTTP2_NGHTTP2_H 1'                 => 1,
+    '#define HAVE_NGHTTP3_NGHTTP3_H 1'                 => 1,
+    '#define HAVE_NGTCP2_NGTCP2_CRYPTO_H 1'            => 1,
+    '#define HAVE_NGTCP2_NGTCP2_H 1'                   => 1,
+    '#define HAVE_OPENSSL_CRYPTO_H 1'                  => 1,
+    '#define HAVE_OPENSSL_ERR_H 1'                     => 1,
+    '#define HAVE_OPENSSL_PEM_H 1'                     => 1,
+    '#define HAVE_OPENSSL_RSA_H 1'                     => 1,
+    '#define HAVE_OPENSSL_SSL_H 1'                     => 1,
+    '#define HAVE_OPENSSL_X509_H 1'                    => 1,
+    '#define HAVE_QUICHE_H 1'                          => 1,
     '#define HAVE_SSL_SET_QUIC_USE_LEGACY_CODEPOINT 1' => 1,
-    '#define HAVE_STDINT_H 1' => 1,
-    '#define HAVE_STDIO_H 1' => 1,
-    '#define HAVE_STDLIB_H 1' => 1,
-    '#define HAVE_STRING_H 1' => 1,
-    '#define HAVE_SYS_XATTR_H 1' => 1,
-    '#define HAVE_UNICODE_UIDNA_H 1' => 1,
-    '#define HAVE_WOLFSSH_SSH_H 1' => 1,
-    '#define HAVE_ZSTD 1' => 1,
-    '#define HAVE_ZSTD_H 1' => 1,
-    '#define LT_OBJDIR ".libs/"' => 1,
-    '#define NEED_LBER_H 1' => 1,
-    '#define PACKAGE "curl"' => 1,
-    '#define PACKAGE_BUGREPORT "a suitable curl mailing list: https://curl.se/mail/"' => 1,
-    '#define PACKAGE_NAME "curl"' => 1,
+    '#define HAVE_STDINT_H 1'                          => 1,
+    '#define HAVE_STDIO_H 1'                           => 1,
+    '#define HAVE_STDLIB_H 1'                          => 1,
+    '#define HAVE_STRING_H 1'                          => 1,
+    '#define HAVE_SYS_XATTR_H 1'                       => 1,
+    '#define HAVE_UNICODE_UIDNA_H 1'                   => 1,
+    '#define HAVE_WOLFSSH_SSH_H 1'                     => 1,
+    '#define HAVE_ZSTD 1'                              => 1,
+    '#define HAVE_ZSTD_H 1'                            => 1,
+    '#define LT_OBJDIR ".libs/"'                       => 1,
+    '#define NEED_LBER_H 1'                            => 1,
+    '#define PACKAGE "curl"'                           => 1,
+'#define PACKAGE_BUGREPORT "a suitable curl mailing list: https://curl.se/mail/"'
+      => 1,
+    '#define PACKAGE_NAME "curl"'     => 1,
     '#define PACKAGE_STRING "curl -"' => 1,
-    '#define PACKAGE_TARNAME "curl"' => 1,
-    '#define PACKAGE_URL ""' => 1,
-    '#define PACKAGE_VERSION "-"' => 1,
-    '#define SIZEOF_LONG_LONG 8' => 1,
-    '#define VERSION "-"' => 1,
-    '#define _FILE_OFFSET_BITS 64' => 1,
-    );
+    '#define PACKAGE_TARNAME "curl"'  => 1,
+    '#define PACKAGE_URL ""'          => 1,
+    '#define PACKAGE_VERSION "-"'     => 1,
+    '#define SIZEOF_LONG_LONG 8'      => 1,
+    '#define VERSION "-"'             => 1,
+    '#define _FILE_OFFSET_BITS 64'    => 1,
```

Comment:
Is there a way to disable enforcing vertical aligments? I think in this particular
case (and in most cases by experience) it makes code harder to edit and can
reduce legibility of diffs, where changing a single line can trigger re-aligning
the whole block.

It seems something better left for human decision and not align by default,
as in most existing code.